### DISCUSSION TITLE
Not working "Add new keyword"...

### DISCUSSION THREAD

**Discussion by 15KR** [marcel-goldammer/obsidian-keyword-highlighter#25] [Q&A]
![Not working](https://github.com/marcel-goldammer/obsidian-keyword-highlighter/assets/167404364/a2787122-99a7-4d04-a35f-e040814cbc7c)

https://github.com/marcel-goldammer/obsidian-keyword-highlighter/assets/167404364/15fcae40-6595-47b6-924d-00088efb2ac5

It's not working. 😭😭😭

**Comment by marcel-goldammer** [comment]
Can you please open the Developer Console, navigate to the Console tab and check if there are any errors, when you click the `Add new keyword` button?

You can access the Developer Console by pressing `cmd + option + i` on Mac or `ctrl + shift + i` on Windows and Linux.

**Comment by 15KR** [comment]
[image: 스크린샷 2024-04-18 오후 8.32.17.png]

An error like this....

I'm using it really well. Thank you for your work.


2024년 4월 18일 (목) 오후 8:28, Marcel Goldammer ***@***.***>님이 작성:

**Reply by marcel-goldammer** [reply]
Sadly I cannot see that image

**Comment by 15KR** [comment]
I added the file below.

**Reply by marcel-goldammer** [reply]
Sorry, I am a litte confused. I only see the gif and mp4 in your first post. But I cannot see your other posted image (png format). Does the error still exsist?

**Comment by marcel-goldammer** [comment]
I created a new release [1.3.1](https://github.com/marcel-goldammer/obsidian-keyword-highlighter/releases/tag/1.3.1). Please update the plugin and check if it fixes your problem.
